src/shared/utils/analytics.ts

```
@@ -0,0 +1,280 @@
+ * - 이벤트명: user_action
+ * - 파라미터: ab_variant, action, ...params, timestamp
+ * - 목적: 다양한 사용자 액션 추적 (A/B 그룹별 비교)
+ *
+ * @param variant - A/B 테스트 그룹 ('single' 또는 'multiple')
+ * @param action - 액션 이름 (예: 'button_click', 'page_scroll', 'form_submit')
+ * @param params - 추가 파라미터 (선택사항)
+ */
+export const logUserAction = (
+  variant: ImageGenerationVariant,
+  action: string,
+  params?: Record<string, unknown>
```

ESLint

[error] Insert `,`

(prettier/prettier)